apps/desktop-wallet/src/pages/UnlockedWallet/OverviewPage/AmountsOverviewPanel.tsx

```diff
@@ -95,15 +94,15 @@ const AmountsOverviewPanel: FC<AmountsOverviewPanelProps> = ({ className, addres
         <Balances>
           <BalancesRow>
             <BalancesColumn>
-              <Today>{date ? dayjs(date).format('DD/MM/YYYY') : t('Value today')}</Today>
-              {isPriceLoading || showBalancesSkeletonLoader ? (
+              <Today>{date ? dayjs(date).format('DD/MM/YYYY') : t('Value today (ALPH)')}</Today>
+              {arePricesLoading || showBalancesSkeletonLoader ? (
```

Comment:
Optimally, would should care whether the prices of ALL tokens has been loaded before displaying the ALPH one, right?